core/computed/viewport-meta.js

```diff
@@ -35,7 +35,16 @@ class ViewportMeta {
     }
 
     const viewportProps = parsedProps.validProperties;
-    const isMobileOptimized = Boolean(viewportProps.width || viewportProps['initial-scale']);
+    const initialScale = viewportProps['initial-scale'];
+    if (initialScale && initialScale < 1) {
+      return {
+        hasViewportTag: true,
+        isMobileOptimized: false,
+        parserWarnings: warnings,
```

Comment:
in case this fails here, should we add a warning indicating the value of `initial-scale` is invalid?

Reply:
Nah, I think those warnings are for if the parser encountered an error. The signal to the user here will be the audit failing.